### DISCUSSION TITLE
Pull down refresh never completes

### DISCUSSION THREAD

**Discussion by codejedi365** [community/community#78083] [Mobile]
This happens on the notification page and the primary page. It could happen other places but I have not been able to force a page refresh at all today (in line with the new update).

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.141.0 (116980603)
Device: iPhone 12 (
iOS 17.1.2)
</details>

**Comment by workgroupengineering** [comment]
I have same issue on Android

![Screenshot_20231207_083719_GitHub.jpg](https://github.com/community/community/assets/12531229/d914925c-c325-4eae-b592-ae4a8653eee5)

![Screenshot_20231207_083846_GitHub.jpg](https://github.com/community/community/assets/12531229/fa4305e8-5931-441c-be1e-bcb46ed4a23e)

**Reply by armin-joellenbeck** [reply]
I have the same problen on Android for an empty notifiction page.

Formerly a message appears on the notification page that there are no notifications.

**Comment by codejedi365** [comment]
Today, when I opened the app, my notifications were updated (basically on initial page load) but the force refresh is still broken.  I did not receive a notification banner though for those new items, at least I don't remember one.

**Comment by iskae** [comment]
Hi @codejedi365 👋 

Thank you for reaching out and reporting this ❤️ 

We already have a PR up for iOS for fixing this and I will be investigating it on the Android side 🙇 Please stay tuned for the future releases 😊 

Cheers!

**Reply by codejedi365** [reply]
All fixed for my iPhone! Thank you!
